src/commands/appdistribution-testers-add.ts

```diff
@@ -5,13 +5,24 @@ import { AppDistributionClient } from "../appdistribution/client";
 import { getEmails, getProjectName } from "../appdistribution/options-parser-util";
 
 export const command = new Command("appdistribution:testers:add [emails...]")
-  .description("add testers to project")
+  .description("add testers to project (and possibly group)")
   .option("--file <file>", "a path to a file containing a list of tester emails to be added")
+  .option(
+    "--group-alias <group-alias>",
+    "if specified, the testers are added to the group identified by this alias"
+  )
   .before(requireAuth)
   .action(async (emails: string[], options?: any) => {
     const projectName = await getProjectName(options);
     const appDistroClient = new AppDistributionClient();
     const emailsToAdd = getEmails(emails, options.file);
     utils.logBullet(`Adding ${emailsToAdd.length} testers to project`);
     await appDistroClient.addTesters(projectName, emailsToAdd);
+    if (options.groupAlias) {
+      utils.logBullet(`Adding ${emailsToAdd.length} testers to group`);
```

Comment:
nit: Consider changing the plurality of `testers` depending on the length of the emails, like https://github.com/firebase/firebase-tools/blob/f5671f1d3e28a786b15f0af99efa1a3796df5513/src/commands/ext-configure.ts#L145

Reply:
I'm too lazy - there are too many places to change